end_to_end/tpu/test_decode_load_quantized_ckpt.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+dry_run=false
+model='llama2-7b'
+
+while getopts "nm:" opt
```

Comment:
I didn't realize we already had this pattern of bash inputs in MaxText tests... I would prefer to be consistent with the majority of other tests but this is also fine